### ISSUE TITLE
Possible bug on status desktop

### ISSUE THREAD

**Issue by jpzuerich** [status-im/status-desktop#4341]
Dear Dev-Team
Once I've signed in to status desktop and then close the app, I cannot login again, it says something like 'this process is used by another app'... I close the process or service in TaskManager and then I can startup again.
Also sending money did not go through and did not tell me that something went wrong. It just doesnt do it. With the Android version it worked then fine, but the balance on the desktop version stayed the same.
Below attached error msg comes once in a while, I haven't read it and don't know if it's always the same, but I thought I send it to you, maybe it helps you to further develop this app. In case you want me to send you more error messages, just let me know...

![Status Error](https://user-images.githubusercontent.com/22144488/147488469-d877727a-590f-4d91-8685-df2f9c6b8e4a.png)
![Status Process](https://user-images.githubusercontent.com/22144488/147488470-0420fc2d-13f8-4235-a739-cd4ffcd27dc0.png)

I hope I could be of use to the team, br

**Comment by jpzuerich** [comment]
Oh, btw, I am NOT Jenkins!

**Comment by jrainville** [comment]
Hi @jpzuerich. which version were you using? When you say "not jenkins", you mean you build it yourself? If so, which commit hash? Otherwise, if it is a release build, which version?
Thanks

**Comment by jpzuerich** [comment]
Thanks for your reply, I'm using version 0.3.0.
With 'not jenkins' I mean: based on the errormsg it shows a path with /user/jenkins... I only know /user/public or /user/my name. But the path /user/jenkins should not be on my system, unless I do have a user on my system that is called jenkins. Hence I assumed, when you're reading this attached error message, that this message refers to a user called jenkins, who is doing something 'not the way Status expects him to'. But there is no user called Jenkins on my Windows10 environment.

Just when I went to read the version number, there came again this error message, so I did a screenshot with the version number included... hope this helps
![Screenshot 2022-01-07 222757](https://user-images.githubusercontent.com/22144488/148610326-9bac60a9-f678-4a73-be9d-645d3fb9cfe6.png)

**Comment by jpzuerich** [comment]
Hi Jonathan

Thanks for your reply, I'm using version 0.3.0.
With 'not jenkins' I mean: based on the errormsg it shows a path with
/user/jenkins... I only know /user/public or /user/my name. But the path
/user/jenkins should not be on my system, unless I do have a user on my
system that is called jenkins. Hence I assumed, when you're reading this
attached error message, that this message refers to a user called jenkins,
who is doing something 'not the way Status expects him to'. But there is no
user called Jenkins on my Windows10 environment.

Just when I went to read the version number, there came again this error
message, so I did a screenshot with the version number included... hope
this helps
